packages/design-picker/src/components/style.scss

```diff
@@ -439,6 +439,13 @@
 
 // layout with categories
 .design-picker--has-categories {
+	.responsive-toolbar-group__swipe {
+		.responsive-toolbar-group__swipe-list {
+			padding: 0;
+			margin-left: -8px;
```

Comment:
It have to be `margin: 0 -8px`. Otherwise, the last element will look weird.

![image](https://user-images.githubusercontent.com/13596067/183328281-ef76c62a-5632-4a31-b45e-88633c8edfbb.png)

Reply:
Also, could you help to update the `padding-bottom` as it's not aligned with the Figma? It might be `24px`.

![image](https://user-images.githubusercontent.com/13596067/183328717-d1015149-9150-401d-ac60-aa2fb2e28969.png)

Reply:
@arthur791004, I the latest update:
- Update the padding bottom to match the design
- I change the solution from shifting the whole list to edit the padding of the last and first items, so can see the result below
https://user-images.githubusercontent.com/10071857/183363546-af7abcf1-1f23-42a4-82d3-413179bf9f8c.mp4